src/Exception/ExceptionConverter.php

```diff
@@ -43,6 +30,20 @@ public static function handleDownloadException(\Throwable $e): void
             ));
         }
 
+        self::handleCommonException($e);
+    }
+
+    private static function handleCommonException(\Throwable $e): void
```

Comment:
tady by si měl ještě odchytávat interface `FlysystemException `

Reply:
- `handleCopyFilesException` a `handlePrepareToDownloadException` by som nespajal, kedze je to to tak lepsie rozsiritelne, ... do buducna moze byt nejaka Exception, ktora patri iba k jednemu z tych dvoch, ... alebo budeme potrebovat inu spravu ak sa jedna o `copy` a inu ak sa jedna o `download`, ... tak su riesene aj vsetky testy.
- interface `FlysystemException` neexistuje, ... tak bol navrhnuty PR do tej externej kniznice, ale oni si to pred mergom este upravili ... tomu zodpoveda `League\Flysystem\FilesystemException`, ... takze je to IMHO spravne

Reply:
https://github.com/thephpleague/flysystem/pull/1070#issuecomment-570794787

Reply:
https://github.com/thephpleague/flysystem/commit/c63d001e8fbd0e795e4e84d52bb56eab0028c29a